src/cascadia/TerminalSettings/IControlSettings.idl

```diff
@@ -12,6 +12,14 @@ namespace Microsoft.Terminal.Settings
         Hidden
     };
 
+    enum ImageStretchMode

```

Comment:
Do you need to create your own enum here? could `BackgroundImageStretchMode` be of type `...Media.Strech`? Then you wouldn't need to static_cast on TermControl.cpp:208 either

Reply:
There's no compelling reason to do so, I suppose; I was following the precedent in the Profile code, but in retrospect utilizing the existing enum when the intent is to exhaustively implement all values seems the more reasonable option.

Switched over to the existing enum on the most-recent commit.